VideoLocker/src/main/java/org/edx/mobile/task/GetFollowingThreadListTask.java

```diff
@@ -6,33 +6,31 @@
 import org.edx.mobile.discussion.DiscussionPostsSort;
 import org.edx.mobile.discussion.TopicThreads;
 import org.edx.mobile.http.RetroHttpException;
-import org.edx.mobile.view.adapters.IPagination;
 
 public abstract class GetFollowingThreadListTask extends Task<TopicThreads> {
-    String courseId;
-    DiscussionPostsSort orderBy;
-    DiscussionPostsFilter filter;
-    IPagination pagination;
+    private static final int PAGE_SIZE = 20;
```

Comment:
Maybe; that's not something I intend to change as part of this PR though.

As an aside... one could make the argument that the client should choose the page size since the amount of items you'd want depends on the size of the screen and how the items are rendered to fill up that screen.